.travis.yml

```diff
@@ -11,39 +11,13 @@ notifications:
 
 # Setup anaconda
 before_install:
-  #- sudo add-apt-repository ppa:octave/stable -y
-  #- sudo apt-get update -q
-  #- sudo apt-get install gdebi -y
-  #- wget https://github.com/jgm/pandoc/releases/download/1.17.0.2/pandoc-1.17.0.2-1-amd64.deb
-  #- sudo gdebi pandoc-1.17.0.2-1-amd64.deb -n
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-  - chmod +x miniconda.sh
-  - ./miniconda.sh -b
-  - export PATH=/home/travis/miniconda3/bin:$PATH
-  - conda update --yes conda
-  # The next couple lines fix a crash with multiprocessing on Travis and are not specific to using Miniconda
-  - sudo rm -rf /dev/shm
-  - sudo ln -s /run/shm /dev/shm
-# Install packages
+  - pip install --upgrade pip
```

Comment:
Explanation necessary in the commit description why you have decided not to use Anaconda.

Reply:
this can definitely be separated into a separe PR. the reason is that it make the installation script faster and simpler. since this is a nonbinary package, it should also be equivalent.

Reply:
Let's split it out then. Matti sounds pretty busy, we need to get going with the fixes. My opinion is that if miniconda is enough to pass the tests, I am good with it (either in this PR or a separate).

Reply:
@fgregg another advantage of use of anaconda is that it is a common Python distribution, which pweave warrants to support

Reply:
Per SO answer: 

> The difference is that miniconda is just shipping the repository management system. So when you install it there is just the management system without packages. Whereas with Anaconda, it is like a distribution with some built in packages.

So if we install all deps with conda, they should be equivalent.

Reply:
happy to split it out.